project-backend/Providers/JobsProvider/JobsProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using project_backend.Models.User;
+using project_backend.Providers.UserProvider;
+using project_backend.Repos;
+using System.Linq;
+namespace project_backend.Providers.JobsProvider
+{
+    public class JobsProvider : IJobsProvider
+    {
+        private readonly DatabaseContext _dbContext;
+
+        public JobsProvider(DatabaseContext databaseContext)
+        {
+            _dbContext = databaseContext;
+        }
+
+        public JobDAO AddJob(string name, string description, int authorId)
+        {
+            JobDAO newJob = new JobDAO
+            {
+                Name = name,
+                Description = description,
+                AuthorId = authorId,
+                DateOfPost = System.DateTime.Now
```

Comment:
You can add in the DatabaseContext class, an onModelCreating override method in which you can specify any kind of constraint you would also specify on an actual database:
https://docs.microsoft.com/en-us/ef/core/modeling/generated-properties?tabs=data-annotations

Reply:
Yeah, tried that but I couldn't get it to work. MySql and SLQ Server seem to have different timestamp functions so it would be annoying to keep replacing anyway.

Reply:
True. Anyways, the time should be set correctly on the machine I will be hosting it, so using the local time of the machine should be fine. I'll have to make sure of that.